### PR TITLE
refactor: improve NodeOutputField and NodeInputField components

### PR DESCRIPTION
**Pull request by anovazzi1**
Refactored the NodeOutputField component in the GenericNode file to improve code organization and readability. Also refactored the NodeInputField component in the GenericNode file to remove unnecessary code and improve performance. These changes enhance the overall maintainability and efficiency of the codebase.

### CODE REVIEW

File: src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx [hunk 1]
```diff
@@ -85,12 +85,12 @@ export default function NodeOutputField({
 
   function handleOutputWShortcut() {
     if (!displayOutputPreview || unknownOutput) return;
-    if (isThereModal() && !openOutputModal) return;
+    console.log("output shortcut");
```
Comment (lucaseduoli) [inline src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx:88]: remove this

File: src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx [hunk 2]
```diff
@@ -85,12 +85,12 @@ export default function NodeOutputField({
...
     if (selected) {
       setOpenOutputModal((state) => !state);
     }
   }
-
+  console.log("outputName", outputName);
```
Comment (lucaseduoli) [inline src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx:93]: remove this

**Review by lucaseduoli — CHANGES_REQUESTED** [review]
LGTM, just remove console logs

File: src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx (hunk 1 above)
Comment (anovazzi1) [reply src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx]: done

File: src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx (hunk 2 above)
Comment (anovazzi1) [reply src/frontend/src/CustomNodes/GenericNode/components/NodeOutputfield/index.tsx]: done

**Review by lucaseduoli — APPROVED** [review]
LGTM

**Pull request merged**